eng/pipelines/templates/steps/analyze.yml

```diff
@@ -24,26 +25,42 @@ steps:
       ArtifactName: "PackageInfo"
       ArtifactPath: "$(Build.ArtifactStagingDirectory)/PackageInfo"
       CustomCondition: and(succeeded(), ${{ parameters.IsSdkLibrary }})
-
   - template: /eng/common/pipelines/templates/steps/set-default-branch.yml
 
-  - task: Powershell@2
```

Comment:
Is there a reason we cannot use what is under eng/common?

Reply:
I've not compared but ideally we would delete the go specific version and use the common one. @praveenkuttappan is there a reason we need a non-common version?

Reply:
The reason is that go does not publish an artifact like the other languages, so this script contains logic to create the `.gosources` before calling the common `Create-APIView.ps1` script.